.env.template

```diff
@@ -14,8 +14,6 @@ SUGCON_INDIA_HOST_INTERNAL_URI=http://sugconindia:3000
 SUGCON_NA_HOST=sugconna.xmcloudcm.localhost
 SUGCON_NA_HOST_INTERNAL_URI=http://sugconna:3000
 RENDERING_HOST_INTERNAL_URI=http://sugconanz:3000
-SUGCON2024_EU_HOST=sugconeu2024.xmcloudcm.localhost
```

Comment:
Why are the SUGCON 2024 Env Variables removed (also a bit down?)

Reply:
This was because we had both SUGCON_EU_HOST and SUGCON2024_EU_HOST, where there is now only one EU site. I dropped the 2024 named version to align with the naming of the other SUGCON sites

Reply:
Same with the CDP related params as well.